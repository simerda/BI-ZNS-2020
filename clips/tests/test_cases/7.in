ne
6
5
ne
ano
