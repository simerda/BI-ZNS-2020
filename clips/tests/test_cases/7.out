CLIPS> Mas figurku ve hre (ano/ne)?
Co jsi naposled hodil (1/2/3/4/5/6)?
Nasad novou figurku do hry.
Hazej znovu!
Co jsi naposled hodil (1/2/3/4/5/6)?
Mas nejakou figurku, ktere zbyva N poli do domecku? (ano / ne)
Mas nejakou figurku, ktera ma N poli pred sebou cizi figurku? (ano / ne)
Tahni tou figurkou.
